src/python/WMCore/WMRuntime/Bootstrap.py

```diff
@@ -206,6 +208,108 @@ def loadTask(job):
         raise BootstrapException(msg)
     return task
 
+def createWMRuntimeJson(outputPath, jobTypes=("Production", "Processing")):
+    """
+    Create a json with runtime information in the following form below:
+    {
+     "workflow_type: "Stepchain",  # string with the request type
+     "number_of_cmsRuns": 2,  # an integer >= 0
+     "worker_arch": "X86_64",  # string with the architecture of the worker node
+     "worker_os": "rhel9",  # string with the OS of the worker node
+     "job_type": "Production", # string with job type information: Production/Processing, Merge, LogCollect, etc.
+     "cmsRun_params": [{
+         "step": "cmsRun1",  # string with the relevant step
+         "keek_output": boolean saying whether we keep output or not,
```

Comment:
Looks like a typo, should the key be `keep_output`?

Reply:
Yes Matti, it should be `keep_output`. I will soon pick a couple of examples from workflows that ran this week and share those here with you.

Reply:
Thanks Alan.